docs/source/tutorial/typesfuns.rst

```diff
@@ -499,14 +497,14 @@ names, ``n`` and ``a``, which are not declared explicitly. These are
     index : forall a, n . Fin n -> Vect n a -> a
 
 Implicit arguments, given with the ``forall`` declaration,
-are not given in applications of ``index``; their values can be
+are not supplied in applications of ``index``; their values can be
```

Comment:
Maybe "provided", also fine to keep as "given", but "supplied" seems off in terms of what I've typically seen people use in this context.

Reply:
"provided" would suggest that the arguments are made available to the user by someone else (e.g., the compiler or library).

"given" is fine, except that it's slightly repetitive (and thus slightly off-putting), because it was just used in the line above.

Reply:
I tried another variant. What do you think?